src/docs/product/performance/web-vitals.mdx

```diff
@@ -35,18 +34,22 @@ If you notice a region of interest on any of the histograms, click and drag over
 
 If you wish to see all of the data available, open the dropdown and click "View All". You will likely see extreme outliers when you click "View All". You can click and drag over an area to zoom in for a more detailed view.
 
-## Web Vitals
+## Thresholds
 
-[Web Vitals](https://web.dev/vitals/) are a set of metrics defined by Google to measure render time, response time, and the sum of shifting rendering processes. Each data point provides insights about the overall performance of your application.
+Google's "Bad", "Needs improvement", and "Good" levels are used to classify data points into red, yellow, and green for the corresponding Web Vitals.
 
-![Visualization of Web Vitals](./diagram-transaction-vitals.png)
+The vertical line for each Web Vital is the 75th percentile of the observed data points. In other words, 25% of the recorded values exceed that amount. If the value exceeds the Google Web Vitals "Bad" threshold, the Web Vital status is marked as "Fail".
```

Comment:
Does this need to explicitly call out "Google's Web Vitals Threshold"?

Reply:
in my first reading i was confused about who defined this threshold. i think maybe wording it as "If the value exceeds Google's "Bad" threshold" would be better?